vendor/pagarme/ecommerce-module-core/src/Kernel/ValueObjects/Installment.php

```diff
@@ -48,7 +48,7 @@ public function __construct($times, $baseTotal, $interest)
     private function setTimes($times)
     {
         $newTimes = intval($times);
-        if ($newTimes < 1 || $newTimes > 24) {
+        if ($newTimes < 0 || $newTimes > 24) {
             throw new InvalidParamException(
                 "A installment times should be set between 1 and 24!",
```

Comment:
Corrigir **string** para:

`"A installment times should be set between 0 and 24!"`

Lembrando de corrigir também a tradução :)

Reply:
Resolvido.

Reply:
Sobre a tradução, não existe tradução nesse quesito, é apenas para Log.